storage/s3.go

```diff
@@ -1049,7 +1049,7 @@ func (s *S3) doDelete(ctx context.Context, chunk chunk, resultch chan *Object) {
 
 		resultch <- &Object{
 			URL: url,
-			Err: fmt.Errorf(aws.StringValue(e.Message)),
+			Err: fmt.Errorf("%v", aws.StringValue(e.Message)),
```

Comment:
Why is this change necessary? Do you use a lint tool other than the `make check` tooling?

Reply:
Hi @igungor , I get this error when trying `make check`, 

```
❯ go version
go version go1.22.6 darwin/arm64
❯ staticcheck -version
staticcheck 2024.1.1 (0.5.1)
❯ make check
storage/s3.go:1052:9: printf-style function with dynamic format string and no further arguments should use print-style function instead (SA1006)
make: *** [staticcheck] Error 1
```

Reply:
👍